docker-compose.example.yml

```diff
@@ -10,23 +10,36 @@ services:
     environment:
       DATABASE_URL: postgresql://van_dam:password@db/van_dam?pool=5
       SECRET_KEY_BASE: a_nice_long_random_string
+      REDIS_URL: redis://redis:6379/1
```

Comment:
Nice catch here - Redis isn't actually in active use (yet) so it's not thrown an error, but it wouldn't have worked without this.